### PR TITLE
Add QSO continuum estimation module

### PR DESCRIPTION
**Pull request by nhmc**
This adds a new module to estimate the continuum level for high- (UVES) and moderate- (ESI/MagE/Xshooter) resolution spectra of a QSO. See the notebook example for usage. It's not perfect, but pretty good IMO.

It also fixes some tests which were failing after PR #20 was merged, adds a new module in analysis `interp.py` which has Akima splines, and adds a new UVES spectrum which tests the continuum fitting.

### CODE REVIEW

**Comment by nhmc** [comment]
(Note this PR is also the first step to including the interactive continuum fitting)

**Comment by profxj** [comment]
I updated the numbers in test_absline_anly.py
in PR #27.  But you can leave the tol edits if you wish.

File: linetools/utils.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+""" Module for general utilities which don't belong an a sub-package.
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+
+def between(a, vmin, vmax):
+    """ Return a boolean array True where vmin <= a < vmax.
+
```
Comment (profxj) [inline linetools/utils.py]: Please add Parameters and Returns
comments here.

File: linetools/tests/test_absline_anly.py [hunk 2]
```diff
@@ -50,11 +50,13 @@ def test_boxew_absline():
...
 
-    np.testing.assert_allclose(ew.value, 0.990466303607004)
+    np.testing.assert_allclose(ew.value, 0.990466303607004, rtol=1e-2)
     assert ew.unit == u.AA
 
     abslin.measure_restew() 
-    np.testing.assert_allclose(ew.value, 0.990466303607004/(1+abslin.attrib['z']))
+    np.testing.assert_allclose(ew.value,
+                               0.990466303607004/(1+abslin.attrib['z']),
+                               rtol=1e-2)
```
Comment (profxj) [inline linetools/tests/test_absline_anly.py]: I fixed the numbers in PR#27 which was related to the
new UM184 files.  You can remove the rtol parameters.

File: linetools/utils.py (hunk 1 above)
Comment (nhmc) [inline linetools/utils.py]: Ok, done.

**Comment by nhmc** [comment]
Merging. Travis is now running, but I haven't configured the tests properly. I will try to fix this soon.

**Pull request merged**